web/src/ferrostar-core.ts

```diff
@@ -88,6 +124,15 @@ export class FerrostarCore extends LitElement {
       bearing: 0,
       zoom: 18,
     });
+
+    if (this.useIntegratedSearchBox) {
+      this.searchBox = new MapLibreSearchControl({
+        onResultSelected: (feature) => {
+          this.startNavigationFromSearch(feature.geometry.coordinates);
+        },
+      });
+      this.map.addControl(this.searchBox, "bottom-left");
```

Comment:
I'd suggest this going in the top left instead (convention; that's where most such UIs put this).